docs/design/modules/digital_twin_graph/README.md

```diff
@@ -33,6 +34,16 @@ The Managed Subscriber Service is an optional service that provides integration
 
 ![Component Diagram](diagrams/digital_twin_graph_component.svg)
 
+## <a name="identifiers">Identifiers</a>
+
+The Digital Twin Graph will use a variety of identifiers. We will discuss the purpose of each.
+
+The provider ID is the identifier for a Digital Twin Provider.
```

Comment:
I would add something like "A provider ID may be associated with multiple instance IDs" to illustrate the hierarchy and show that providers and entities do not have to be one-to-one

Reply:
Fixed.